src/magicbox/SuggestionsManager.ts

```diff
@@ -221,6 +224,36 @@ export class SuggestionsManager {
     });
   }
 
+  private get querySuggestPreviewComponent() {
+    const querySuggestPreviewElement: HTMLElement = $$(this.root).find(`.${Component.computeCssClassNameForType('QuerySuggestPreview')}`);
+    if (!querySuggestPreviewElement) {
+      return;
+    }
+    const querySuggestPreview = <QuerySuggestPreview>Component.get(querySuggestPreviewElement);
+    if (!(querySuggestPreview.options.numberOfPreviewResults > 0)) {
+      return;
+    }
+
+    return querySuggestPreview;
+  }
+
+  private setUpForResultPreview(suggestions: Dom): Dom {
```

Comment:
Setup is not 2 words, maybe something like initPreviewForSuggestions would be a bit better